src/main/webapp/templates/keywords.html

```diff
@@ -1,15 +1,37 @@
-<h1>{{title}}</h1>
-<form id="keyword-form" action="/keyword" method="POST">
-  <textarea 
-    name="keyword-sentence" 
-    placeholder="Enter your tweet or interests here">Black Lives Matter and COVID-19.</textarea>
-  <br/>
-  <input type="submit" />
-</form>
-<div id="keyword">
-  <ul>
-    {{#keywords}}
-    <li>{{term}}</li>
-    {{/keywords}}
-  </ul>
+<div id="keywords"> 
+  {{#keyterms}}
```

Comment:
This should be looping through "keywords", not "keyterms" (I think in general we should keep it as "keywords")